src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const normalizedTitle = movie.title.toLowerCase();
+    const normalizedDescription = movie.description.toLowerCase();
+    const normalizedQuery = query.trim().toLowerCase();
```

Comment:
It would be nice to delete unnecessary spaces between words in the search query. If I search 'love actually' (with two spaces) I will not get results. You can use something like this:
const normalizedQuery = query.toLowerCase().split(' ').filter(Boolean).join(' ');